concrete/src/Updater/Migrations/Migrations/Version20180820205800.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Concrete\Core\Updater\Migrations\Migrations;
+
+use Concrete\Core\Updater\Migrations\AbstractMigration;
+use Concrete\Core\Updater\Migrations\RepeatableMigrationInterface;
+
+class Version20180820205800 extends AbstractMigration implements RepeatableMigrationInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function upgradeDatabase()
+    {
+        $this->refreshBlockType('express_entry_list');
+    }
+}
```

Comment:
This line-ending should probably be fixed

Reply:
I think if you click this link, you'll see what I mean: https://github.com/concrete5/concrete5/pull/6971/files#diff-fb26234b780da062675b4595855abcc6R17